pkg/manager/manager.go

```diff
@@ -139,6 +139,23 @@ type Options struct {
 	// value only if you know what you are doing. Defaults to 10 hours if unset.
 	// there will a 10 percent jitter between the SyncPeriod of all controllers
 	// so that all controllers will not send list requests simultaneously.
+	//
+	// A period sync happens for two reasons:
+	// 1. To insure against a bug in the controller that causes an object to not
+	// be requeued, when it otherwise should be requeued.
+	// 2. To insure against a bug in controller-runtime, or its dependencies,
```

Comment:
This sounds like there was a known bug. I think it is very unlikely that there is a bug in that in either controller-runtime or the underlying kube api machinery

Reply:
```suggestion
	// 2. To insure against an unknown bug in controller-runtime, or its dependencies,
```